HyperV/scripts/export-eventlog.ps1

```diff
@@ -19,13 +21,34 @@ function dumpeventlog($path){
 
 function exporteventlog($path){
 
-	Get-Eventlog -list | ForEach-Object {
+	Get-Eventlog -list | Where-Object { $_.Entries -ne '0' } | ForEach-Object {
 		$logfilename = "eventlog_" + $_.LogDisplayName + ".txt"
```

Comment:
We can drop completely the txt export, it's supposed to be replaced by the html one.